.ci.yaml

```diff
@@ -54,6 +65,26 @@ targets:
       target_file: mac_build_all_packages.yaml
       channel: stable
 
+  - name: Mac_arm64 custom_package_tests master
+    bringup: true # New configuration
```

Comment:
Did you try a `led` run with this?  Suspect you'll hit https://github.com/flutter/flutter/issues/113231

Reply:
Technically you can copy an existing config and edit it go/flutter-luci-recipes#testing-a-local-change-for-a-new-builderrecipe.  However I've never had the patience to do that.
Usually I remove bringup and presubmit in the PR and let it run in presubmit, and then revert that before merging.  ci.yaml validation would fail though.  You can normally be sneaky and use an existing test and just say swap out `target_file` to have it run in presubmit, but you have the added complication of there being no existing `Mac_arm64` builders.

Reply:
Yeah, that sounds about like what I've done in the past; this was to bootstrap me to something where I could play around with things in presubmit in a follow-up PR.

But since we have known ARM breakage I'll hold off on this, and maybe explore doing an Intel version instead. Thanks for the pointer to that issue!